integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.service.ts

```diff
@@ -0,0 +1,183 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { ActiveCartFacade, Cart } from '@spartacus/cart/base/root';
+import {
+  CheckoutBillingAddressFacade,
+  CheckoutDeliveryAddressFacade,
+  CheckoutPaymentFacade,
+} from '@spartacus/checkout/base/root';
+import {
+  Address,
+  Country,
+  GlobalMessageService,
+  GlobalMessageType,
+  HttpErrorModel,
+  UserPaymentService,
+} from '@spartacus/core';
+import {
+  BehaviorSubject,
+  combineLatest,
+  EMPTY,
+  Observable,
+  throwError,
+} from 'rxjs';
+import {
+  catchError,
+  filter,
+  finalize,
+  map,
+  shareReplay,
+  switchMap,
+  take,
+  tap,
+} from 'rxjs/operators';
+import { OpfCheckoutPaymentWrapperService } from '../opf-checkout-payment-wrapper';
+
+@Injectable()
+export class OpfCheckoutBillingAddressFormService {
+  protected readonly billingAddressSub = new BehaviorSubject<
+    Address | undefined
+  >(undefined);
+  protected readonly isLoadingAddressSub = new BehaviorSubject(false);
```

Comment:
[MEDIUM]

`isLoadingAddressSub` vs `_$isLoadingAddress?`?

Let's avoid opinionated and ambiguous abbreviations like `Sub`. At first glance, it can be mistakenly interpreted to mean `Subscription`.

Note: analogical remark applies to other property(ies) in this class

Reply:
[OUTDATED]